docs/content/policy-reference.md

```diff
@@ -893,6 +895,12 @@ cached response is in fact still fresh. If the server responds with a `200` (`OK
 with the new response. On a `304` (`Not Modified`) server response, `http.send` will update the headers in cached response with
 their corresponding values in the `304` response.
 
+The `force_cache` field can be used to override the cache directives defined by the server. This field is used in
+conjunction with the `force_cache_duration_seconds` field. If `force_cache` is `true`, then `force_cache_duration_seconds`
+**must** be specified and `http.send` will use this value to check the freshness of the cached response.
```

Comment:
If `force_cache_duration_seconds` must be specified, do we need `force_cache`?

Reply:
The idea would be to allow users to control more of the caching directives and we are starting with just one for now. So in the future you could have more than just `force_cache_duration_seconds` and then we can use `force_cache` to override `cache` parameter while not making `force_cache_duration_seconds ` mandatory.